src/unix/fs.c

```diff
@@ -916,25 +916,16 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
     off = req->off;
 
 #ifdef __linux__
-    {
-      static int copy_file_range_support = 1;
-
-      if (copy_file_range_support) {
```

Comment:
Since `copy_file_range_support` is set to `1`, wouldn't the part inside the `if` block run anyway?

Reply:
Being `copy_file_range_support` static, it's initialized to `1` only the first time this code block runs. The following times will have whatever value it was set later on. For example if the syscall wasn't supported, `copy_file_range_support` will be set to 0 the next time `uv__fs_sendfile()` is called.

Reply:
Ah, thanks for telling! Sorry, I overlooked `static`.